src/Pages/AdminTaskTable/components/AdminTaskFilters.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { fireEvent, render } from '@testing-library/react';
+import AdminTaskFilters from './AdminTaskFilters';
+
+jest.mock('../../../middleware/AppContext', () => ({
+  useAppContext: () => ({}),
+}));
+
+jest.mock('../../../services/useDebounce', () => (value) => value);
+jest.mock('react-query');
+
+it('Render loading state (disabled)', async () => {
+  const { getByRole } = render(
+    <AdminTaskFilters
+      isLoading={true}
+      setFilterData={() => null}
+      filterData={{
+        statuses: [],
+        accountId: '',
+        orgId: '',
+      }}
+    />,
+  );
+
+  const filterInput = getByRole('textbox');
+  expect(filterInput).toHaveAttribute('disabled');
+});
+
+it('Select a filter of each type and ensure chips are present', async () => {
+  const { queryByText, getByRole, getByLabelText } = render(
+    <AdminTaskFilters
+      setFilterData={() => null}
+      filterData={{
+        statuses: [],
+        accountId: '',
+        orgId: '',
+      }}
+    />,
+  );
+
+  // Enter an account ID item
+  const accountIdInput = getByRole('textbox');
+  expect(accountIdInput).not.toHaveAttribute('disabled');
+  fireEvent.change(accountIdInput, { target: { value: '11593016' } });
+
+  const optionMenu = getByLabelText('Options menu');
+  fireEvent.click(optionMenu);
+
+  const orgIdOption = queryByText('Org ID') as Element;
+  expect(orgIdOption).toBeInTheDocument();
+  fireEvent.click(orgIdOption);
+
+  // Enter an org ID item
+  const orgIdInput = getByRole('textbox');
+  expect(orgIdInput).not.toHaveAttribute('disabled');
+  fireEvent.change(orgIdInput, { target: { value: '13446804' } });
```

Comment:
Try to get in the habit of setting reused variables in tests. 
A typo here can cause needless time debugging.